docs/timeseries/masking.rst

```diff
@@ -0,0 +1,52 @@
+.. _timeseries-masking:
+
+Masking values in time series
+*****************************
+
+.. |Table| replace:: :class:`~astropy.table.Table`
+
+Masking values is done in the same way as for |Table| objects (see
+`Masking and missing values
+<http://docs.astropy.org/en/stable/table/masking.html>`_). The easiest way
```

Comment:
Why hardcode URL here instead of using Intersphinx? This can be confusing if people are looking at dev doc since it links to stable regardless.

Reply:
Because this was developed in a separate package originally :) Fixed now.